SolarTherm/Data/Optics/surround/50MWe/4555c%/883K/1000kWpm2/24dSM/isp_optimised/100%phi_100%HT_100%Arecv_optics.motab

```diff
@@ -0,0 +1,125 @@
+#1

```

Comment:
this filename should have the spaces (%20) removed.

Reply:
so these files are all SolarPILOT output that you're using in your models as inputs for your packed bed storage systems. perhaps they could be put into a folder with a better name, so that their relevance to a specific model is clear? or is there a readme in this or a parent folder to clarify that?